shared/chat/conversation/messages/wrapper/chat-send.js

```diff
@@ -75,12 +75,23 @@ class SendIndicator extends React.Component<Props, State> {
     }
   }
 
+  _restart() {
+    if (shownEncryptingSet.has(this.props.id)) {
+      shownEncryptingSet.delete(this.props.id)
+    }
+    this._setVisible(true)
```

Comment:
this `_setVisible(true)` call gets overridden by the following `_setStatus` call, so you can just remove it (`visible` should already be true I think)